graf3d/eve7/src/REveDataClasses.cxx

```diff
@@ -313,7 +313,7 @@ void REveDataColumn::SetExpressionAndType(const std::string& expr, FieldType_e t
      << ") = [](" << icls->GetName() << "* p){" << icls->GetName() << " &i=*p; return (" << fExpression.Data()
      << "); }";
 
-   // printf("%s\n", s.Data());
+   printf("%s\n", s.str().c_str());
```

Comment:
Do you want to have this active for now? (Just a question - either is fine with me.)

Reply:
It does not need to be active print. Will comment it. Thanks!